monkey/monkey_island/cc/services/initialize.py

```diff
@@ -90,6 +96,13 @@ def _build_agent_binary_repository():
     return agent_binary_repository
 
 
+def _build_mongo_credentials_repository():
+    mongo = MongoClient(MONGO_URL, serverSelectionTimeoutMS=100)
```

Comment:
Add this object to the DI container so future repositories can use the same connection.